--- Running polars.Expr.str.contains#7NN/containsNN, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'polars.Expr.str.contains#7NN'), ('notebook', 'containsNN'), ('text', '  File "containsNN.py", line 3, in <module>\n    default_match = a.str.contains(\'AA\',  False, strict=True)\nTypeError: contains() takes 2 positional arguments but 3 positional arguments (and 1 keyword-only argument) were given\n')])
  File "containsNN.py", line 3, in <module>
Error inducing API short name: contains, line no: 3
Error inducing API fqn: UNKNOWN, line no: 3
*** TypeError: PLACEHOLDER
Repair Action: arg_name
Key Params: []
Prev Params: None
Prev Repair Action: None
Line No: 3
Prev Line No: 0
GitHub Candidates:
[]
KP:[]
EXPP: []
APIDoc Candidates:
[]
All Solution Candidates:
[]
[INFO] This case cannot be fixed!


real	0m0.388s
user	0m0.356s
sys	0m0.033s
